libr/include/r_print.h

```diff
@@ -46,7 +46,7 @@ typedef struct r_print_t {
 	int datezone;
 	int (*write)(const unsigned char *buf, int len);
 	void (*cb_printf)(const char *str, ...);
-	const char *(*cb_color)(int idx, int last, bool bg);
+	char *(*cb_color)(int idx, int last, bool bg);
```

Comment:
libr/cons/pal.c:r_cons_rainbow_get   returns an allocated string (tl;dr, but its is not constant).